content/en/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough.md

```diff
@@ -202,8 +202,8 @@ the only other supported resource metric is memory.  These resources do not chan
 to cluster, and should always be available, as long as Heapster is deployed.
 
 You can also specify resource metrics in terms of direct values, instead of as percentages of the
-requested value.  To do so, use the `target` type of `AverageValue`, and thus set `target.averageValue`
-field instead of the `target.averageUtilization` field.
+requested value, by using the `target` type of `AverageValue` instead of `averageUtilization`. This sets the
+`target.averageValue` field instead of the `target.averageUtilization` field.
```

Comment:
this is wrong -- it's a discriminated/tagged union (`target` is the tag, `averageUtilization` and friends are the value).  You need to set `target` to `AverageValue` *and then* set `target.averageValue` yourself.  This wording implies the latter gets set automatically.  The original intention was

> if you set `target` to `Xyz`, you must also set the `xyz` field